src/analysis/utils/notebook-lockers.ts

```diff
@@ -0,0 +1,28 @@
+import _ from 'lodash/fp';
+import { notebookLockHash } from 'src/analysis/utils/file-utils';
+import { Ajax } from 'src/libs/ajax';
+import { GoogleWorkspace, hasAccessLevel } from 'src/workspaces/utils';
+
+export const findPotentialNotebookLockers = async (workspace: GoogleWorkspace): Promise<{ [key: string]: string }> => {
```

Comment:
also removing a circular dependency trap by breaking this out from src/analysis/utils/file-utils.ts